tests/expectations/std_lib.compat.c

```diff
@@ -3,17 +3,11 @@
 #include <stdint.h>
 #include <stdlib.h>
 
-typedef struct Option_i32 Option_i32;
-
-typedef struct Result_i32__String Result_i32__String;
-
-typedef struct Vec_String Vec_String;
-
 #ifdef __cplusplus
 extern "C" {
 #endif // __cplusplus
 
-void root(const Vec_String *a, const Option_i32 *b, const Result_i32__String *c);
+void root(const Vec<String> *a, const Option<int32_t> *b, const Result<int32_t, String> *c);
```

Comment:
This is a bug, something here is wrong, this is not supposed to change behavior, right?